### PR TITLE
Shutdown

### PR DESCRIPTION
**Pull request by g4s8**
#297 - shutdown take and deploy command in `.rultor.yml` + `@todo`

### CODE REVIEW

File: .rultor.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@ install: |-
   java -version
 deploy:
   script: |
+    curl -f https://www.0crat.com/shutdown > /dev/null
```
Comment (yegor256) [inline .rultor.yml]: @g4s8 we don't use `deploy`. Instead, we are releasing via `release` command. I guess, this line should stay right before the `git push` we are doing to Heroku

**Comment by yegor256** [comment]
@g4s8 looks good, see above

**Comment by g4s8** [comment]
@yegor256 All my pull requests lose my fork when I changed repository to public:
scrot-2018-01-24-13 28 03
So I submited new PR: #456

File: .rultor.yml (hunk 1 above)
Comment (g4s8) [reply .rultor.yml]: @yegor256 fixed in new PR

**Comment by 0crat** [comment]
Order was successfully finished: +15 points just awarded to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx), total is [+270](https://www.0crat.com/u/fabriciofx)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://datum.zerocracy.com/pages/policy.html#28): +15 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256), total is [+8045](https://www.0crat.com/u/yegor256)

**Comment by 0crat** [comment]
The job [#440](https://github.com/zerocracy/farm/issues/440) is now out of scope

**Pull request closed without merging**